config/root-config.bat.in

```diff
@@ -21,6 +21,10 @@ if "%1"=="" (
    exit /b 1
 )
 
+for %%i in ("%~dp0..") do set "folder=%%~fi"
```

Comment:
I am not familiar with the bat syntax.  What does this do?

Reply:
the parent directory of the location of `root-config.bat` (i.e. what should be `%ROOTSYS%`)

Reply:
Thanks